kolibri/utils/server.py

```diff
@@ -651,6 +694,9 @@ def __init__(
         pid_plugin = PIDPlugin(self)
         pid_plugin.subscribe()
 
+        systemd_plugin = SystemdNotifyPlugin(self)
```

Comment:
I would rather do the conditionalization of the plugin here, rather than within the plugin itself, see the example below for the auto reloader.

So I'd suggest getting the env var value here, then conditionally creating and subscribing the plugin.

Can also pass the env var value as an arg to the SystemdNotifyPlugin `__init__` method.

Reply:
Done in a fixup commit (which should be squashed before this is merged; I kept it separate for ease of re-review).

I abstracted the check into a class method on `SystemdNotifyPlugin` to stop users of it having to worry about the implementation details of what counts as a valid `NOTIFY_SOCKET` setting.